scripts/src/java/org/oppia/android/scripts/maven/BUILD.bazel

```
@@ -0,0 +1,30 @@
+"""
+Libraries corresponding to attributing third-party library Copyright licenses in Oppia-android.
```

```suggestion
Libraries corresponding to attributing third-party library copyright licenses in Oppia Android.
```

For consistency in capitalization.

---

Done.

---

Verified.